src/core/hle/kernel/k_page_table.h

```diff
@@ -64,6 +64,15 @@ class KPageTable final : NonCopyable {
         return page_table_impl;
     }
 
+    constexpr bool Contains(VAddr addr) {
```

Comment:
```suggestion
    bool Contains(VAddr addr) const {
```

This can't be used in a non-constexpr context